cms/djangoapps/contentstore/views/course.py

```diff
@@ -810,6 +810,18 @@ def _create_or_rerun_course(request):
                     status=400
                 )
 
+        # Allow user to create the course only if they belong to the organisation
+        # This flag doesn't apply to Global Staff and Superusers
+        if settings.FEATURES.get('RESTRICT_COURSE_CREATION_TO_ORG_ROLES', False):
+            has_org_permission = has_studio_write_access(request.user, None, org)
+            if not has_org_permission:
+                log.exception("User does not have the permission to create course \
+                    in this organization. User: {} Org: {} Course: {}".format(request.user.id, org, course))
```

Comment:
@farhaanbukhsh, it is better to add line breaks after the opening parenthesis and use implicit string concatenation of adjacent strings, no?